src/utils/contract-event-listeners.ts

```diff
@@ -41,15 +41,20 @@ function getCreateSarcoHandler(networkContext: NetworkContext) {
       ethWallet.address
     );
 
-    const block = await ethWallet.provider.getBlock(event.blockNumber);
-    const creationDate = getDateFromTimestamp(block.timestamp);
+    let creationDate;
+    try {
+      const block = await ethWallet.provider.getBlock(event.blockNumber);
+      creationDate = getDateFromTimestamp(block.timestamp);
+    } catch (error) {
+      archLogger.warn(`unable to get block timestamp: ${error}`)
+    }
 
     inMemoryStore.get(networkContext.chainId)!.sarcophagi.push({
       id: sarcoId,
       resurrectionTime: scheduledResurrectionTime,
       perSecondFee: archaeologist.diggingFeePerSecond,
       cursedAmount: archaeologist.curseFee,
-      creationDate,
+      creationDate: creationDate || Date.now(),
```

Comment:
couldn't find any evidence this creationDate has any meaning in the codebase (meaning it's harmless to set to Date.now()) -- but need to verify 100%